packages/turf-dissolve/index.js

```diff
@@ -29,118 +24,49 @@ import { closestGreaterNumber } from "./lib/get-closest";
  * //addToMap
  * var addToMap = [features, dissolved]
  */
-function dissolve(featureCollection, options) {
+function dissolve(fc, options) {
   // Optional parameters
   options = options || {};
   if (!isObject(options)) throw new Error("options is invalid");
-  var propertyName = options.propertyName;
+  const propertyName = options.propertyName;
```

Comment:
Build is failing because `const` isn't es6 and we aren't doing any transpiling in non-ts modules as part of the build.